### ISSUE TITLE
Highly insecure

### ISSUE THREAD

**Issue by timowevel1** [achmadqomarudin/MeetingApp#2]
Hey bro,

please help me and don't ignore it. Could you please guide how to make it more secure in terms of the database?

Thanks.

**Comment by achmadqomarudin** [comment]
Of course, I'll give you documentation about Firestore Security and Youtube to understanding about the rules :
- https://firebase.google.com/docs/firestore/security/get-started
- https://firebase.google.com/docs/firestore/security/insecure-rules
- https://www.youtube.com/watch?v=DjI4vdmZ7UU

**Comment by timowevel1** [comment]
I think you dont understand it. I cant just change the Rules, you also have to change the code. Because when you e.g. restrict the Access to the Database, your App cant access it anymore, so you have to find another solution.

**Comment by timowevel1** [comment]
Why you just dont make a 5 min tutorial when it is so easy like you say to change it?
